cegs_portal/static/css/project.css

```diff
@@ -1613,6 +1613,11 @@ input[type="submit"], input[type="button"], button {
   line-height: 2rem;
 }
 
+.text-lg {
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+}
+
```

Comment:
Doesn't this need to be added to project.css.tw?

Reply:
I regenerated the files when running the local server but no changes were detected in project.css.tw